src/app/components/LinkedData/index.tsx

```diff
@@ -111,12 +111,37 @@ const LinkedData = ({
 
   const hasByline = !!bylineLinkedData;
 
-  const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
-    bylineLinkedData || {};
+  const places: string[] = [];
 
-  const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+  const getAuthorTagging = (data: BylineLinkedData | null) => {
+    const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
+      data || {};
 
-  const locationCreated = { '@place': location };
+    const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+    if (location) {
+      if (!places.includes(location)) {
+        places.push(location);
```

Comment:
Let's combine this into a single line

```suggestion
      if (location && !places.includes(location)) {
        places.push(location);
```